app/app-dev.html

```diff
@@ -9,27 +9,13 @@
     <div class="draggable-bar"></div>
     <div id="root"></div>
     <script>
-      if (typeof process === 'object') {
-        const scripts = [];
+      window.__dirname = electron.__dirname;
+      window.__filename = electron.__filename;
 
-        if (process.env.NODE_ENV === 'development') {
-          // Dynamically insert the DLL script in development env in the
-          // renderer process
-          scripts.push('../dll/renderer.dev.dll.js');
-        }
-        if (process.env.START_HOT) {
-          // Dynamically insert the bundled app script in the renderer process
-          const port = process.env.PORT || 1212;
-          scripts.push(`http://localhost:${port}/dist/renderer.dev.js`);
-        } else {
-          scripts.push('./dist/renderer.prod.js');
-        }
-
-        if (scripts.length) {
-          document.write(
-            scripts.map(script => `<script defer src="${script}"><\/script>`).join('')
-          );
-        }
+      if (electron.scriptsToLoad.length) {
+        document.write(
+          electron.scriptsToLoad.map(script => `<script defer src="${script}"><\/script>`).join('')
+        );
```

Comment:
Interestingly, I wonder if you could XSS someone running this wallet in dev mode, with a malicious `PORT` env variable? Not saying this is cause for concern - in prod this is safe.

Reply:
This is from the boilerplate we used. `contextIsolation` should prevent any XSS here though, as the preload file, that defines `electron` as a global (may rename to app or something), is running in a separate context to the renderer.